xalia/Win32/HwndProvider.cs

```diff
@@ -365,44 +365,64 @@ private void PollChildren()
 
             var child_hwnds = EnumImmediateChildWindows(Hwnd);
 
-            // First remove any existing children that are missing or out of order
+            // Remove or ignore any existing children
+            HashSet<IntPtr> seen_children = new HashSet<IntPtr>();
             int i = 0;

```

Comment:
Is this a copy paste error? I don't see `seen_children` used here.

Reply:
Odd, VS usually warns me about unused things. But yes, that shouldn't be there as I'm pretty sure win32 can't give me duplicate child hwnds.